src/PluginMediaStreamRenderer.swift

```diff
@@ -27,16 +27,16 @@ class PluginMediaStreamRenderer : NSObject, RTCEAGLVideoViewDelegate {
 		// It's placed over the elementView.
 		self.videoView = RTCEAGLVideoView()
 
-		self.webView.addSubview(self.elementView)
-		self.webView.bringSubviewToFront(self.elementView)
-
 		self.elementView.userInteractionEnabled = false
 		self.elementView.hidden = true
-		self.elementView.backgroundColor = UIColor.blackColor()
+		self.elementView.backgroundColor = UIColor.clearColor()
```

Comment:
I mean that if the webview has a transparent background and the video element doesn't cover it completely, you'd see the (transparent) background, which is weird whitish. I agree it's better to have black bars on the video, or if the user wants they can set object-fit to get rid of them.